src/store/options.js

```diff
@@ -29,6 +29,14 @@ const Options = {
   [store.connect]: {
     async get() {
       const { options = {} } = await chrome.storage.local.get(['options']);
+
+      // Set default value for keys, which type does no match the current one
+      Object.entries(options).forEach(([key, value]) => {
+        if (typeof value !== typeof Options[key]) {
+          delete options[key];
```

Comment:
Do you think it makes sense to log a warning here? It should not trigger often, right?

The logic is fine, but it might be that someone later writes code that mixes up strings (`"42"`) and numbers (`42`). We should not do that on purpose, of course, but logging a warning would help to catch these kind of bugs.

Reply:
You would have to change the structure of the `Options` object - it is a definition of the model, so I assume, that you will not change that by mistake. If then in some view, you would set a value `42` instead of `"42"` it is fine - the store keeps the correct type and cast the value.

The only case is if you update the model definition keeping the same key, but with a different type. I think it is very unlikely to be a problem on production, but as we pushed to TestFlight a version, which has changed then, so it becomes a problem here (and it is very hard to clear the extension persistent data on a real device).

I can add here a `console.warn()`, but it rather not help much ;) but why not :)